src/uwtools/scheduler.py

```diff
@@ -1,346 +1,372 @@
 """
-Job Scheduling.
+Support for HPC schedulers.
```

Comment:
This module has been updated to cooperate with the `uwtools.drivers.driver` and `uwtools.drivers.fv3` modules.